src/Reflection/ReflectionClass.php

```diff
@@ -236,7 +244,10 @@ public function getExtensionName() : ?string
      */
     private function getAllMethods() : array
     {
-        return array_merge(
+        $traitAliases     = $this->getTraitAliases();
+        $traitPrecedences = $this->getTraitPrecedences();
+
+        return array_filter(array_merge(
```

Comment:
would put the `array_filter()` only around the logic that fetches methods from traits